src/cr/cube/stripe.py

```diff
@@ -239,6 +272,7 @@ def __init__(self, rows_dimension, means, unweighted_counts):
 
     @lazyproperty
     def rows(self):
+        """Sequence of _MeansMrStripeRow for each valid element in rows-dimension."""
```

Comment:
Ok, I see a pattern now, is there a reason why you used the `Sequence` instead of `tuple`?